internal/store/postgres/job/job_repository.go

```diff
@@ -248,6 +250,79 @@ func (j JobRepository) preCheckUpdate(ctx context.Context, jobEntity *job.Job) e
 	return nil
 }
 
+func getJobDiff(storageSpecOld *Spec, newJobEntity *job.Job) (ChangeLog, error) {
+	toIgnore := map[string]struct{}{
+		"ID":        {},
+		"CreatedAt": {},
+		"UpdatedAt": {},
+	}
+	var changelog []Change
+	storageSpecNew, err := toStorageSpec(newJobEntity)
+	if err != nil {
+		return changelog, err
+	}
+
+	diff, err := utils.GetDiffs(*storageSpecOld, *storageSpecNew, nil)
+	if err != nil {
+		return changelog, err
+	}
+
+	for _, d := range diff {
+		if _, ok := toIgnore[d.Field]; !ok {
+			if d.IsDiffTypeUnified() {
```

Comment:
Can we decide at library level to use unified or not

Reply:
sure